### ISSUE TITLE
JPEG files not ingested with the local Ollama recommended setup.

### ISSUE THREAD

**Issue by llFllLllll** [zylon-ai/private-gpt#1922]
contrary to other files types like pdf or pptx, images are not ingested and no error popup. Any idea? tks

**Comment by psychopatz** [comment]
of course it will not read images since it just extracts the text from the document files, your best bet is to ocr the images before ingesting. privategpt doesnt support llava or ocr yet

**Comment by mrepetto-certx** [comment]
I agree with you, but then https://docs.privategpt.dev/manual/document-management/ingestion should be revised; otherwise, it is misleading.

**Comment by kprice-hii** [comment]
Interesting, at least when I try to upload an image I don't get any errors. Pptx files on the other hand always throw a previously mentioned error about needing torch, etc. despite it being installed.
